releases.md

```diff
@@ -152,8 +153,7 @@ Older releases are EOL and available on [GitHub][tekton-pipeline-releases].
 [v0-43-1-docs]: https://github.com/tektoncd/pipeline/tree/v0.43.1/docs#tekton-pipelines
 [v0-43-2-docs]: https://github.com/tektoncd/pipeline/tree/v0.43.2/docs#tekton-pipelines
 [v0-42-0-docs]: https://github.com/tektoncd/pipeline/tree/v0.42.0/docs#tekton-pipelines
-[v0-41-1-docs]: https://github.com/tektoncd/pipeline/tree/v0.41.1/docs#tekton-pipelines
-[v0-41-0-docs]: https://github.com/tektoncd/pipeline/tree/v0.41.0/docs#tekton-pipelines
```

Comment:
Any reason to remove the other bugfix link ? (if yes, we might want to do it for all bugfix releases)

Reply:
I am noticing the pattern of maintaining links to the latest patch based on the release notes guidelines. Yup, I checked and confirmed we have been following the same guidelines for other bugfix releases before creating this PR. No action needed here. Thanks!